apps/liteprotocoltester/lpt_supervisor.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+
+import os
+import time
+from subprocess import Popen
+import sys
+
+def load_env(file_path):
+  env_vars = {}
+  with open(file_path) as f:
+    for line in f:
+      if line.strip() and not line.startswith('#'):
+        key, value = line.strip().split('=', 1)
+        env_vars[key] = value
+  return env_vars
+
+def start_tester_node():
+  role = sys.argv[1]
+  env = os.environ.copy()
+  for key, value in env_vars.items():
+    if key not in env:
+      env[key] = value
+  script_cmd = ['sh', '/usr/bin/run_tester_node_at_infra.sh', '/usr/bin/liteprotocoltester', role]
+  process = Popen(script_cmd, env=env)
+  return process
+
+if __name__ == "__main__":
+  if len(sys.argv) < 2 or sys.argv[1] not in ["RECEIVER", "SENDER"]:
+    print("Error: First argument must be either 'RECEIVER' or 'SENDER'")
+    sys.exit(1)
+
+  env_file = '/usr/bin/infra.env'
```

Comment:
Why are you hardcoding path to the env vars file? Wouldn't it be simpler to just check if the env var is defined and read it from the code?